napari/utils/colormaps/colormap.py

```diff
@@ -657,6 +657,10 @@ def _cast_labels_data_to_texture_dtype_auto(
         converted = np.where(
             data_arr == colormap.selection, selection_in_texture, dtype.type(0)
         )
+    elif data.size < 1000:
```

Comment:
is this threshold justified by benchmarks?

Reply:
Ten times bigger than the default brush size. No benchamrks. No idea how to benchmark it. Currently need to be at least 2 (to not use threads for mapping array of size 1).